bool lock_rsp
bool unlock_rsp
bool open_gripper
bool close_gripper
